www/css/coolhole.css

```diff
@@ -398,17 +398,107 @@
 }
 
 /* CoolPoints User Styling */
-#coinWrapper {
+#cpModal {
+  padding: 0;
+}
+#cpModalContent {
+  padding-top: 5px;
+}
+#cpModalBody {
+  padding: 0;
+}
+#cpModalBody > .container-fluid {
+  padding-left: 18px;
+  padding-right: 18px;
+}
+#cpModalHeader {
   display: flex;
+  padding-top: 5px;
+  padding-bottom: 5px;
+  padding-left: 18px;
+  padding-right: 18px;
+  align-items: center;
+}
+#cpModalHeader > button.close {
+    color: rgb(246, 246, 246);
+    font-size: 24px;
+    opacity: 100%;
+}
```

Comment:
nit: already said it above but I'd like to see us apply this to the style so it's on every modal, not just this one

Reply:
also inconsistent tabbing